docs/products/postgresql/howto/pgbouncer-stats-datadog.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Expose PgBouncer statistics to Datadog
+sidebar_label: PgBouncer stats in Datadog
+---
+
+Connect PgBouncer with Datadog to display [PgBouncer statistics](/docs/products/postgresql/howto/pgbouncer-stats) on the Datadog platform.
+
+## Prerequisites
+
+- Your service plan is Startup or higher.
+- [Datadog metrics integration](/docs/integrations/datadog/datadog-metrics) is
+  enabled on your service.
+- You applied outstanding service maintenance updates affecting the Datadog metrics
+  integration.
+
+## Expose PgBouncer statistics
```

Comment:
I recommend "Enable monitoring" to drop the term "statistics" as well as to keep it consistent with "Enable monitoring" in [Database monitoring with Datadog :: Enable monitoring](https://aiven.io/docs/products/postgresql/howto/monitor-database-with-datadog#enable-monitoring)